### ISSUE TITLE
Add vpc and subnet tagging for EKS

### ISSUE THREAD

**Issue by atsikham** [epiphany-platform/m-aws-basic-infrastructure#5]
**Is your feature request related to a problem? Please describe.**
To be able to use created **VPC** and **subnets** for EKS, it's necessary to add tags according to [documentation](https://docs.aws.amazon.com/eks/latest/userguide/network_reqs.html#vpc-tagging). Check [example](https://github.com/epiphany-platform/epiphany/blob/feature/eks-cluster-creation/docs/design-docs/eks/terraform-scripts/aws-vpc.tf).
**Describe the solution you'd like**
Add tagging to resources.
**Describe alternatives you've considered**
Tag resources after creation in EKS module, but It seems not reasonable to do so, moreover it's [not allowed](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ec2_tag) to apply for bunch of resources.
**Additional context**
No.

**Comment by atsikham** [comment]
This task is postponed and probably will be implemented in EKS module in the scope of #1636

**Comment by atsikham** [comment]
I'm closing this issue, as it can be done in eks module not in a complex way with terraform's [ec2_tag](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ec2_tag) resource. For doing this multiple times something similar can be used:

```hcl
resource "aws_ec2_tag" "xxx" {
  count = length(var.tags)
  ...
}
```
